frontend/packages/operator-lifecycle-manager/integration-tests/views/operator-hub.view.ts

```diff
@@ -4,7 +4,7 @@ export const operatorModal = $('.pf-c-modal-box');
 export const operatorModalBody = $('[data-test-id="operator-modal-box"]');
 export const operatorModalHeader = $('[data-test-id="operator-modal-header"]');
 export const operatorModalIsLoaded = () =>
-  browser.wait(until.presenceOf(operatorModalBody), 1000).then(() => browser.sleep(500));
+  browser.wait(until.presenceOf(operatorModalBody), 15000).then(() => browser.sleep(5000));
```

Comment:
Do we need the 5 second sleep? That will make the tests run a lot slower locally. If there some other element we can wait to be visible to avoid the hard-coded value?

Reply:
Just tested with original "500" and flake or not, it passed. So, I'm reverting it, though I hope you are okay with "15000" max-wait, as I think it'd help cut down flakes. Let me know if you want to import `BROWSER_WAIT` and use that instead.

Reply:
The sleep is back to 500, and 15000 has been dropped as that'd be the same as default wait. PTAL. Thanks!